### PR TITLE
Update READMEs

### PR DESCRIPTION
**Pull request by jayantk**
The documentation had some duplication between terra and solana and was a little confusing in places. I went through and clarified it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,38 @@
-# Pyth SDK
+# Pyth Network SDK
 
-This repo contains multiple crates for using Pyth Oracle.
-1. Pyth SDK: This crate contains general Pyth structures and interfaces which are consistent across different blockchains.
-2. Pyth SDK Solana: This crate contains methods for reading and parsing Pyth structures from Pyth Solana accounts.
+The Pyth Network Rust SDK provides utilities for reading price feeds from the [pyth.network](https://pyth.network/) oracle in on- and off-chain applications.
```
Comment (jayantk) [inline README.md:3]: moved SDK highlights to the root README from the solana README

File: pyth-sdk-solana/README.md [hunk 2]
```diff
@@ -30,96 +21,51 @@ Pyth Network stores its price feeds in a collection of Solana accounts of variou
...
-let price: PriceFeed = load_price_feed_from_account_info( &price_account_info ).unwrap();
+let price_feed: PriceFeed = load_price_feed_from_account_info( &price_account_info ).unwrap();
+let current_price: Price = price_feed.get_current_price().unwrap();
+println!("price: ({} +- {}) x 10^{}", current_price.price, current_price.conf, current_price.expo);
 ```
 
-#### Off-chain
-To read the price from a price account off-chain in clients, this library provides a `load_price_from_account` method that constructs `Price` struct from Account:
+The `PriceFeed` object returned by `load_price_feed_from_account_info` contains all currently-available pricing information about the product.
+This struct also has some useful functions for manipulating and combining prices; see the [common SDK documentation](../pyth-sdk) for more details.
```
Comment (jayantk) [inline pyth-sdk-solana/README.md:47]: moved the docs about PriceFeed into a the pyth-sdk readme

File: pyth-sdk-terra/README.md [hunk 3]
```diff
@@ -1,83 +1,44 @@
...
-println!("0.1 BTC and 0.05 ETH are worth: ({} +- {}) x 10^{} USD",
-         basket_price.price, basket_price.conf, basket_price.expo);
+// Pyth network testnet contract address
+pyth_contract_addr: string = "terra1hdc8q4ejy82kd9w7wj389dlul9z5zz9a36jflh";
+// Price feed id for BTC/USD on testnet
+price_feed_id: string = "0xf9c0172ba10dfa4d19088d94f5bf61d3b54d5bd7483a322a982e1373ee8ea31b";
+
+let price_feed: PriceFeed = query_price_feed(deps.querier, pyth_contract_addr, price_feed_id)?.price_feed;
+let current_price: Price = price_feed.get_current_price().ok_or(StdError::not_found("price is not currently available"))?;
+println!("current BTC/USD price: ({} +- {}) x 10^{}", current_price.price, current_price.conf, current_price.expo);
```
Comment (jayantk) [inline pyth-sdk-terra/README.md:29]: made this example complete so that the reader can basically copy-paste it into their code

File: README.md [hunk 4]
```diff
@@ -1,19 +1,38 @@
...
+### Creating a Release
 
-To  release new versions of these packages, perform the following steps within the crate being released:
+To release a new version of any of these crates, perform the following steps within the crate being released:
 
 1. Increment the version number in `Cargo.toml`.
    You may use a version number with a `-beta.x` suffix such as `0.0.1-beta.0` to create opt-in test versions.
 2. Merge your change into `main` on github.
 3. Create and publish a new github release.
+   This step will trigger a Github Action that publishes a new version of the crate to [crates.io](https://crates.io).
```
Comment (ali-bahjati) [inline README.md:38]: We don't have it yet 😓

**Review by ali-bahjati — APPROVED** [review]
Thanks a lot! 🙇

**Pull request merged**